examples/Manifest.toml

```diff
@@ -169,15 +169,20 @@ version = "0.1.4"
 
 [[deps.ClimaComms]]
 deps = ["CUDA", "StaticArrays"]
-git-tree-sha1 = "93c4c545024821c9cacb7417f00381b2329e76db"
+git-tree-sha1 = "2c7d29daf60f028710b5be57a24f981757703585"
+repo-rev = "main"
```

Comment:
Just an note: this should be changed to a specific version and not the main branch

Reply:
Yes. I was testing before version release.